server/config.go

```diff
@@ -41,6 +41,7 @@ type Config interface {
 	GetDatabase() *DatabaseConfig
 	GetSocial() *SocialConfig
 	GetRuntime() *RuntimeConfig
+	GetMatch() *MatchConfig
```

Comment:
wdyt of calling this config flag `multiplayer`?

Reply:
Debated and decided to stick with `match`.